tests/test_classification_accs.py

```diff
@@ -1,7 +1,10 @@
 import unittest
 import numpy as np
+import sys
 
-from mindsdb_evaluator.accuracy.classification import evaluate_multilabel_accuracy
+sys.path.insert(0, "../mindsdb_evaluator/accuracy")
```

Comment:
Replying to your comment in Slack:

> Due to the new metric not being included yet in the module, the tests had to be run with sys.path mangling - this method worked on my machine but doesn't work in the Github action.

The GitHub actions should work for this branch, because you _have_ included the metric in the module, so this bit is unnecessary. Doing:

> from mindsdb_evaluator.accuracy.classification import evaluate_multilabel_accuracy, evaluate_top_k_accuracy

Should be enough. Looking at the commit history it doesn't seem like you've tried that, could you give it a go?

Reply:
Hello @paxcema, thanks for the feedback! https://github.com/mindsdb/mindsdb_evaluator/pull/16/commits/388ea60ab28692ce47120bdd9ada8fc23806e897 contains the code with the changes you mentioned, which seems to work now :)

(This method didn't work locally though, might be an issue on my system - sorry for the inconvenience.)

Reply:
No worries, glad it's working now! Left one more request and then I think we're ready to merge.